packages/styles/src/mixins/list/_list-byline.scss

```diff
@@ -51,8 +51,14 @@ $fd-list-byline-section-text-color: var(--sapContent_LabelColor) !default;
     }
 
     .#{$block}__notification {
-      @include fd-set-paddings-x(1rem, 0.75rem);
+      @include fd-reset();
```

Comment:
![Screenshot 2022-12-14 at 9 44 05 AM](https://user-images.githubusercontent.com/39598672/207626608-fb5e997c-7dde-4acc-8a33-b1645d9ed6cc.png)

You don't need reset. The reset is coming from the `sap-icon-*` class